web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -215,11 +218,11 @@ describe("<LicenseTask />", () => {
 
     it("does not display error alert when license status is found", async () => {
       renderTask();
-
       mockApi.checkLicenseStatus.mockResolvedValue(generateUserData({}));
       fireEvent.submit(screen.getByTestId("check-status-submit"));
+
       await waitFor(() => {
-        expect(screen.getByText("Pending")).toBeInTheDocument();
+        expect(screen.getByText(Config.licenseSearchTask.completedStatusText)).toBeInTheDocument();
       });
       expect(screen.queryByTestId("error-alert-NOT_FOUND")).not.toBeInTheDocument();
```

Comment:
For this test, nothing has changed, I don't understand why this is changing from pending to completed text: I'd expect the permit status to continue to display pending. If this is the text within the Tag, I think there should be an assertion around the permit status text and probably rename `completedStatusText` so it's more clear that it's text related to the Tag component